lib/lita/message.rb

```diff
@@ -13,6 +13,11 @@ class Message
     # @return [Source] The message source.
     attr_reader :source
 
+    # Adapter specific data.
+    # @return [Hash] The adapter specific data.
+    # @since 4.6.2
+    attr_reader :adapter_data
```

Comment:
No, each object has its own extensions. Just using the same name on each object because they have the same purpose.